react/Image.tsx

```diff
@@ -62,11 +57,17 @@ function Image(props: ImageProps) {
     />
   )
 
+  /**
+   * To understand why we need to check for both newTab and openNewTab
+   * properties, check the Image type definition at './typings/image.d.ts'.
+   */
+  const shouldOpenLinkInNewTab = link?.newTab ?? link?.openNewTab
+
   return link ? (
     <a
       href={formatIOMessage({ id: link.url, intl })}
       rel={link.noFollow ? 'nofollow' : ''}
-      target={link.openNewTab ? '_blank' : ''}
+      target={shouldOpenLinkInNewTab ? '_blank' : ''}
```

Comment:
I don't think you should pass an empty string to target, maybe is better to pass `'_self'` or `undefined`
```suggestion
      target={shouldOpenLinkInNewTab ? '_blank' : undefined}
```